olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -382,6 +382,45 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    */
   bool IsCached(const geo::TileKey& tile) const;
 
+  /**
+   * @brief Protects tile keys from eviction.
+   *
+   * @note You can only protect tiles which data handles are present in the
+   * cache at the time of the call.
+   *
+   * @note Please do not call `Protect` while the `Release` call for the same
+   * catalog and layer is in progress.
+   *
+   * @note Protecting tile keys mean that its data and corresponding quad tree
+   * keys will be added to protected list and stored in cache. This keys will be
+   * removed from LRU, so they could not be evicted. Also they are not expire.
+   * Quad tree stays protected if at least one tile key is protected.
```

Comment:
Move to line 388 (before the note).

```suggestion
   * Protecting tile keys means that its data and corresponding quadtree
   * keys are added to the protected list and stored in the cache. These keys are
   * removed from the LRU cache, so they could not be evicted. Also, they do not expire.
   * The quadtree stays protected if at least one tile key is protected.
```